library/ssl_tls13_generic.c

```diff
@@ -2743,7 +2743,8 @@ void mbedtls_ssl_conf_early_data( mbedtls_ssl_config* conf, int early_data,
 void mbedtls_ssl_conf_signature_algorithms( mbedtls_ssl_config *conf,
                      const int* sig_algs )
 {
-    conf->sig_hashes = sig_algs;
```

Comment:
This definition should also be guarded by `MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED` I think